docs/Rootless.md

```diff
@@ -33,19 +35,32 @@ endif
 ```
 
 - The iOS 14 arm64e ABI mentioned in [arm64e Deployment](arm64e-Deployment.html) is now *required* for the relevant devices
-    - Currently, it's only possible to build for the new ABI on macOS
-        - If you want to maintain support for earlier versions, you can grab the toolchain from an earlier Xcode release as specified in [arm64e Deployment](arm64e-Deployment.html) and switch between it and the newer toolchain as desired by setting the `PREFIX` variable to the older toolchain's bin (i.e., `<xcode-ver>.xctoolchain/usr/bin/`) in your project's makefile
+    - Currently, it's only possible to build for the new ABI on macOS as the necessary `ld64` changes, included in Xcode, have not been made open source
+        - Unfortunately, new toolchain does not support building for the old ABI. If you want to maintain support for earlier versions, you can grab the toolchain from an earlier Xcode release as specified in [arm64e Deployment](arm64e-Deployment.html) and switch between it and the newer toolchain as desired by setting the `PREFIX` variable to the older toolchain's bin (i.e., `<xcode-ver>.xctoolchain/usr/bin/`) in your project's makefile
     - That being said, developers on other platforms can circumvent this *if necessary*:
-        - By adding `oldabi` as a dependency to their package (preferably only as a last resort)
+        - By [using GitHub Actions](https://github.com/p0358/SilentScreenshots/blob/master/.github/workflows/build.yml) to compile their tweaks (free for both public and private repos)
+        - By using a macOS virtual machine
+            - See KVM on Linux and VMware on Windows
+            - If you want to reduce resources used by the VM (after installing Xcode and Theos):
```

Comment:
```suggestion
            - If you'd like to reduce resources used by the VM after installing Xcode and Theos:
```

Reply:
Hmm, doesn't this make it sound more like installing Xcode and Theos increases resource usage without the parentheses? The reason I added that fragment is just so that someone doesn't disable the window server before installing Xcode and stuff

Reply:
Good point. What about
```suggestion
            - After installing the desired tools, you can optimize the VM's system resource usage by:
```